File: MainClass.java
import java.util.Scanner;
public class MainClass{
public static void main(String[] args) {
    Scanner _key = new Scanner(System.in);
String contadorTxt;
double contador;
contadorTxt = "Contador: ";

contador= _key.nextDouble();
System.out.println(contadorTxt);

System.out.println(contador);

contador = 5;

while (contador<=10){
System.out.println(contadorTxt);
System.out.println(contador);
contador = contador+1;

}
do{
System.out.println(contadorTxt);
System.out.println(contador);
contador = contador-1;

} while (contador>=0);

}}

File: MainClass.py
def main():
	string contadorTxt;
	float contador;
	contadorTxt = "Contador: "

	contador = float(input())

	print(contadorTxt)

	print(contador)

	contador = 5

	
	while contador<=10:
		print(contadorTxt)
		print(contador)
		contador = contador+1

		break

	
	while contador>=0:
		print(contadorTxt)
		print(contador)
		contador = contador-1

		break

